kitchen/resources/classify_image.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
```

Comment:
Can we avoid having this file in our repository?  Perhaps download it via the Dockerfile for the demo app?

Reply:
Okay, will do. 

Reminder to self: pass in the `model_dir` argument explicitly

Reply:
Done.